### DISCUSSION TITLE
Two Screens on a MacBook pro 14"

### DISCUSSION THREAD

**Discussion by KoljaL** [MonitorControl/MonitorControl#1109] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Discussion

Hi there and thank you for this great app!

I use it on my MBP (M1) with an adapter for two displays (DELL U2412M, via HDMI->DVI cable):

https://de.targus.com/products/hyperdrive-dual-4k-hdmi-10-in-1-usb-c-hub-hdm1h-gl

But i can only see all properties of one display.   
For the other there ist only the "software control" item. 

Does anyone have experiences with this setup?

Kolja

**Comment by the0neyouseek** [comment]
As these adapters were made for m1 mac (before m1 pro/max/ultra) which could only support one external display at the time, I'm pretty sure the issue comes from the adapter itself but please follow @waydabber recommendations

**Comment by waydabber** [comment]
Yes, the adapter has two HDMI outputs, one of them is native the other one is probably DisplayLink.

Can you post a screenshot of what you see under Preferences/Displays?

Thank you!

**Comment by KoljaL** [comment]
To be exact, my MBP is the 14" version with the M1 pro chip.

Here are the screenshots:

![](https://user-images.githubusercontent.com/55852574/177513206-d3bc7281-1e1e-437e-a53f-4734b8bbec78.png)

![](https://user-images.githubusercontent.com/55852574/177513207-74f5836c-082f-4a7a-8595-7eb1765fb46e.png)

Thanks for your interest :-)

**Comment by waydabber** [comment]
Hi @KoljaL - this seems to be a BetterDisplay related issue, so we should move the issue over there. :)

If you click on the "Dell U2412M", a bunch of additional issue should appear. However from the icon I see that this is a virtual display so the problem is with the DisplayLink adapter.

A small improvement you can achieve is that the latest DisplayLink v1.7 beta driver now allows colortable manipulation. There is a preliminary build of BetterDisplay that will allow a slightly better brightness management experience for these adapters:

https://github.com/waydabber/BetterDisplay/issues/750

This will only work properly only if you use the latest DisplayLink beta driver.

**Comment by waydabber** [comment]
I'll move this to the discussion as there seems to be no issue to be fixed. :)
